OpenOversight/app/auth/views.py

```diff
@@ -64,7 +72,9 @@ def login():
         if user is not None and user.verify_password(form.password.data):
             if user.is_active:
                 login_user(user, form.remember_me.data)
-                return redirect(url_for("main.index"))
+                return redirect(
+                    validate_redirect_url(session.get("next")) or url_for("main.index")
+                )
```

Comment:
Do you mind splitting this out just for readability?
```suggestion
                next = validate_redirect_url(session.get("next"))
                return redirect(next or url_for("main.index"))
```

Reply:
I think next is a reserved keyword but I can make the change in a new commit!

Reply:
D'oh, it is! Some other variable name then 😄